README.md

```diff
@@ -28,11 +27,11 @@ use async_ssh2_tokio::client::{Client, AuthMethod};
 async fn main() -> Result<(), async_ssh2_tokio::Error> {
     // Only ip and password based authentification is implemented.
     // If you need key based authentification, create github issue or contribute.
-    let mut client = Client::new(
+    let mut client = Client::connect(
```

Comment:
This is just a note that this is a breaking change. Therefore this would need to release as 0.5.0.